### PR TITLE
Multiple PBA's - Confirm decision happy

### PR DESCRIPTION
**Pull request by paulhowes-HMCTS**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-3727

### Change description ###
Multiple PBA's - Confirm your decision (Happy Path)

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by tarikdogan-hmcts — APPROVED** [review]

File: src/org-manager/components/new-pbas-confirm/new-pbas-confirm.component.ts [hunk 1]
```diff
@@ -12,7 +15,53 @@ export class NewPBAsConfirmComponent {
...
+      })
+    });
+
+    const payload = {
+      pbaNumbers,
+      orgId: this.org.organisationId
+    }
+
+    this.pbaService.setPBAStatus(payload)
+      .subscribe({
```
Comment (paddybasi) [inline src/org-manager/components/new-pbas-confirm/new-pbas-confirm.component.ts:41]: should we not do any subscription clean up as well?

File: src/org-manager/services/pba.service.ts [hunk 2]
```diff
@@ -17,7 +17,7 @@ export class PbaService {
   }
 
   public setPBAStatus(body: any): Observable<any> {
-    return this.http.put<any>(`${this.pbaStatusUrl}`, body);
+     return this.http.put(`${this.pbaStatusUrl}`, body);
```
Comment (paddybasi) [inline src/org-manager/services/pba.service.ts:20]: rather than remove the `any` type isn't there a way you could strongly type the response?

**Review by paddybasi — APPROVED** [review]

File: src/org-manager/components/new-pbas-confirm/new-pbas-confirm.component.ts (hunk 1 above)
Comment (paulhowes-HMCTS) [reply src/org-manager/components/new-pbas-confirm/new-pbas-confirm.component.ts:43]: Cleanup complete - I've added a subscription property that is unsubscribed in the ngOnDestroy method.

File: src/org-manager/services/pba.service.ts (hunk 2 above)
Comment (paulhowes-HMCTS) [reply src/org-manager/services/pba.service.ts]: Added a ConfirmNewPBAModel Model and strongly typed it in the response.

File: src/org-manager/services/pba.service.ts [hunk 3]
```diff
@@ -17,7 +18,7 @@ export class PbaService {
   }
 
   public setPBAStatus(body: any): Observable<any> {
-    return this.http.put<any>(`${this.pbaStatusUrl}`, body);
+     return this.http.put<ConfirmNewPBAModel>(`${this.pbaStatusUrl}`, body);
```
Comment (paddybasi) [inline src/org-manager/services/pba.service.ts:21]: Great, just need that return type on the function as well

Comment (paulhowes-HMCTS) [reply src/org-manager/services/pba.service.ts:21]: I've changed the return type as well now. Thanks @paddybasi .

**Pull request merged**